src/libostree/ostree-repo.c

```diff
@@ -2094,6 +2094,26 @@ append_remotes_d (OstreeRepo          *self,
 }
 
 gboolean
+ostree_repo_is_created (OstreeRepo    *self,
+                        gboolean      *is_created,
+                        GCancellable  *cancellable,
+                        GError        **error)
+{
+  if (!ostree_repo_open (self, cancellable, error))
+    {
+      if (errno == ENOENT)
```

Comment:
So...this gets a bit confusing.  But we can't rely on the value of `errno` after `ostree_repo_open()`.  We've already translated from `errno -> GError`, so we'd need to look at that.